src/main/java/VASSAL/tools/version/VassalVersionTokenizer.java

```diff
@@ -150,27 +153,25 @@ public int next() throws VersionFormatException {
         break;
 
       case BUILD:
-        {
```

Comment:
This scope shouldn't be removed. It's there to prevent the local variables from being in the same scope as the bodies of other cases.

Reply:
This sub-scope is not necessary. Accessing the local variables from another case body is a compile error. Declaring new variables with the same name is also a compile error.

Reply:
Yes, the point of that scope was to prevent those exact compile errors...